apps/app/src/utils/pageScroll.ts

```diff
@@ -1,13 +1,23 @@
 import "client-only";
 
-const classes = ["overflow-hidden"];
-
-export const lockScroll = () => {
+export const lockScroll = (mobileOnly = false) => {
 	document.body.style.paddingRight = `${window.innerWidth - document.body.offsetWidth}px`;
-	document.body.classList.add(...classes);
+
+	if (mobileOnly) {
+		document.body.classList.add("overflow-hidden", "sm:overflow-auto");
+		return;
+	}
+
+	document.body.classList.add("forceLockScroll");
```

Comment:
Nie jestem pewien czy ten early return ułatwia czytanie tego kodu, nie lepiej byłoby z `else`?
```suggestion
	if (mobileOnly) {
		document.body.classList.add("overflow-hidden", "sm:overflow-auto");
	} else {
		document.body.classList.add("!overflow-hidden");
	}
```